src/app/api/routes/alerts.py

```diff
@@ -58,6 +58,14 @@ async def link_media(payload: AlertMediaId,
             status_code=400,
             detail="Permission denied"
         )
+
+    existing_media = await crud.fetch_one(media, {"id": payload.media_id})
+    if existing_media is None:
+        raise HTTPException(
+            status_code=404,
+            detail="Media does not exist"
+        )
+    existing_alert = dict(**existing_alert)
```

Comment:
isn't this already a dict?

Reply:
same comment:
Nope, it's a record. I got an exception when launching the corresponding test at first.
NB: maybe it's converted implicitly using pydantic when we are in "prod" but the test was failing.